### ISSUE TITLE
My baritone has been crashing for the past 2 months please help

### ISSUE THREAD

**Issue by taterontop** [cabaletta/baritone#2295]
## Some information
Operating system: Windows 10
Java version: latest
Minecraft version: 1.12.2
Baritone version: v1.2.13
Forge mods (if used): No however its happened on impact,kami blue and Inertia

[crash-2020-12-24_18.55.31-client.txt](https://github.com/cabaletta/baritone/files/5740361/crash-2020-12-24_18.55.31-client.txt)

**Comment by taterontop** [comment]
Also im just gonna say but ive deleted the kami,impact,inertia and baritone folder in .minecraft

**Comment by l1ving** [comment]
stop pirating software

**Comment by taterontop** [comment]
It happens when i dont use a client aswell

**Comment by l1ving** [comment]
Yes, because you pirated software. Disable noverify

**Comment by taterontop** [comment]
Oh thanks for telling me

**Comment by taterontop** [comment]
Another bug now. Do i send my latest.log?

**Comment by taterontop** [comment]
Nvm found the problem

**Comment by majorkush** [comment]
I've been having similar issues, I don't run any Pirated software and have the donor edition of aristois, everything was working fine up untell in tryed adding litematica, then my suddenly sel ca command stopped working. after some reading I found out there was an issue with litematica and baritone. so it tryed removing it, and it was giving me some issues.. so I completely uninstalled minecraft and deleted the .minecraft folder, to get a freshs start. after all things were working fine previously. I've tryed switching out the bariton versions a few times before going back to the original (that comes with Aristois*. after many launcher restarts I'm hoping someone there smarter then myself may know... honesly It'ss probably something so stupid i feel bad even typing to you guys. I mean, it was working.. :/ #idiot.... in any case I'm attaching my last crash log. seems fine when I do "sel ca" but soon as i do "sel set stone_bricks" its -saving game. lol.. anyways ty for any help thanks so much for all your time, saved so many ppl so much by sharing baritone. tysm. life is short. hope all is well cheers.
[latest.log](https://github.com/cabaletta/baritone/files/6497991/latest.log)
